### PR TITLE
[Tooltip] v1.0.0

### PR DESCRIPTION
**Pull request by dancingshell**
### Summary

- Update styles to give more room for main content when using sidebar content
- Sidebar content now has max width and truncates to an ellipse
- Removed opacity on error tooltip
- Bump to first major release 🎉

screen shot 2017-12-15 at 12 12 54 pm

### CODE REVIEW

File: tooltip/README.md [hunk 1]
```diff
@@ -4,34 +4,39 @@ A tooltip component.
...
+- Update styles to give more room for main content when using sidebar content
+- Sidebar content now has max width and truncates to an ellipse
+- Bump to first major release 🎉
+
+##### 0.8.0
 - Added sidebar via `sidebarContent` prop
 - Added tests
 
-### 0.7.0
+##### 0.7.0
```
Comment (ben-burlingham) [inline tooltip/README.md]: nb I've been using `###` for major bumps, and `#####` for everything else...sort of confusing i guess

File: tooltip/src/Tooltip.scss [hunk 2]
```diff
@@ -152,8 +156,12 @@ $errorBackgroundColor: rgba(217, 58, 60, 0.9);
...
 
   .sidebarContent {
-    cursor: pointer;
     text-decoration: underline;
     align-self: center;
+    max-width: 60px;
+    white-space: nowrap;
+    overflow: hidden;
+    cursor: pointer;
+    text-overflow: ellipsis;
```
Comment (ben-burlingham) [inline tooltip/src/Tooltip.scss]: nb Alphabetizing saves lives `FN + F5`

File: tooltip/README.md (hunk 1 above)
Comment (dancingshell) [reply tooltip/README.md]: ya i guess the 0 releases were major (at the time)

File: tooltip/src/Tooltip.scss (hunk 2 above)
Comment (nickwmurphy) [reply tooltip/src/Tooltip.scss]: +1

**Pull request merged**